### PR TITLE
Added and improved Dutch translations

### PR DESCRIPTION
**Pull request by aiolos**
Hello,

Great application! I added and improved some translations for Dutch. Hope you like it. 
More will be added later.

### CODE REVIEW

File: MacPass/nl.lproj/Localizable.strings [hunk 1]
```diff
@@ -164,20 +164,20 @@
...
-"ENFORCE_PASSWORD_CHANGE_ALERT_TITLE" = "The database password expired!";
-"ENFORCE_PASSWORD_CHANGE_ALERT_DESCRIPTION" = "You will be unable to save until the password and/or keyfile has been changed.";
+"ENFORCE_PASSWORD_CHANGE_ALERT_TITLE" = "Het databasewachtwoord is verlopen!";
+"ENFORCE_PASSWORD_CHANGE_ALERT_DESCRIPTION" = "U kunt niet opslaan totdat het wachtwoord en/of de keyfile zijn gewijzigd.";
 
-"RECOMMEND_PASSWORD_CHANGE_ALERT_TITLE" = "Please change the database password!";
-"RECOMMEND_PASSWORD_CHANGE_ALERT_DESCRIPTION" = "It is recommended to change the password and/or keyfile.";
-"CHANGE_PASSWORD_WITH_DOTS" = "Change Password…";
+"RECOMMEND_PASSWORD_CHANGE_ALERT_TITLE" = "Wijzig het databasewachtwoord, alstublieft!";
+"RECOMMEND_PASSWORD_CHANGE_ALERT_DESCRIPTION" = "Het is aanbevolgen om het wachtwoord en/of de keyfile te wijzigen.";
```
Comment (MWeesenaar) [inline MacPass/nl.lproj/Localizable.strings]: Aanbevolen*

File: MacPass/nl.lproj/Localizable.strings [hunk 2]
```diff
@@ -164,20 +164,20 @@
...
-"WARNING_ON_SAVE_NO_PASSWORD_OR_KEY_SET_SUGGESTION" = "Please set a password and/or keyfile for this database. Aborting this will undo all changes and lock the document";
+"WARNING_ON_SAVE_NO_PASSWORD_OR_KEY_SET" = "Er is geen wachtwoord of keyfile ingesteld.";
+"WARNING_ON_SAVE_NO_PASSWORD_OR_KEY_SET_SUGGESTION" = "Stel een wachtwoord en/of keyfile in voor deze database. Annuleren zal alle wijzigingen ongedaan maken en het bestand blokkeren";
 
 /* Message displayed when an open file was changed from another application */
-"FILE_CHANGED_BY_OTHERS_MESSAGE_TEXT" = "The database file was modified!";
+"FILE_CHANGED_BY_OTHERS_MESSAGE_TEXT" = "Het databasebestand is gewijzigd!";
 /* Informative text displayed when the file was change form another application */
-"FILE_CHANGED_BY_OTHERS_INFO_TEXT" = "The file loaded is not the same as the on on disk. How do you want to move on?";
+"FILE_CHANGED_BY_OTHERS_INFO_TEXT" = "Het geladen bestand is niet hetzelfde als op schijf. Hoe wilt u verder gaan??";
```
Comment (MWeesenaar) [inline MacPass/nl.lproj/Localizable.strings]: Remove one ?

**Comment by MWeesenaar** [comment]
Added two minor comments. Furthermore good translation. 👍

**Comment by aiolos** [comment]
Thanks, I pushed some fixes.

**Comment by mstarke** [comment]
Thanks for the nice work. I see that you did edit the localizeables file manually and did not include the strings dictionary file. Might I suggest using XLIFF export/import for a streamlined workflow? I switched to this method and found it to be a huge help. If you want to use it, simply select the MacPass project in Xcode, go to Editor > Export for Localisation ans select Dutch. Use a free XLIFF editor (e.g. XLIFFTool) and save the file and then go back to Xcode and use Editor > Import Localisation to incorporate your changes.

**Comment by aiolos** [comment]
Thanks for pointing that out. I already found it a lot of work to find all the files and strings and manually change them. I will try the XLIFF way. (This is also the first time I'm really using XCode and macOS development)

**Comment by mstarke** [comment]
Would you be so kind to coordinate the changes with #714?

**Comment by aiolos** [comment]
I close this PR, #715 is now the one to use.

**Pull request closed without merging**